### PR TITLE
FE-640 Reward-boosts

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -52,7 +52,8 @@ module.exports = {
 				  "rewards/cell-capacity",
 				  "rewards/hardware-class",
 				  "rewards/rewards-claiming",
-				  "rewards/rewards-troubleshooting"
+				  "rewards/rewards-troubleshooting",
+				  "rewards/reward-boosts"
```
Comment (PavlosTze) [inline sidebars.js:56]: Let's move this new boosts secton above "Rewards Claiming" please.

File: docs/rewards/reward-boosts.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+id: reward-boosts
+title: Rewards Boosts
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:3]: Rewards -> Reward

File: docs/rewards/reward-boosts.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_label: Rewards Boosts
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:4]: Rewards -> Reward

File: docs/rewards/reward-boosts.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+slug: /project/rewards-boosts
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:5]: rewards-boosts -> reward-boosts

File: docs/rewards/reward-boosts.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+## Overview
+In addition to the **Base Rewards**, a device already gets for contributing to weather data during the regular daily distribution. Learn more about the daily rewards [Reward Mechanism (RM) ](/reward-mechanism).
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:12]: I think a sentence is missing here before "In addition" e.g. "Reward boosts are rewards......"

File: docs/rewards/reward-boosts.mdx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 
+
+So far the following **Reward Boosts** are implemented:
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:16]: Hmmm not true, Beta Rewards is almost implemented, but Cell Boost is **not**, it's just the next boost down in the pipeline. So let's not say "implemented" here as it's a wrong term.

File: docs/rewards/reward-boosts.mdx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+### Beta Rewards
+
+According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation) 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:20]: a comma is needed before the "3M $WXM"

File: docs/rewards/reward-boosts.mdx [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.
+
+Calculation:
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:24]: Calculation --> #### Calculation

File: docs/rewards/reward-boosts.mdx [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+The following calculations will take place for each station, the day before the Token Launch:
+
+`RewardableBetaHours[s]`.
+This is the total number of hours the station provided valid data during the Beta period.
+A rewardable hour is an hour during which the station sent at least one valid weather data point.
+
+`StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`
+This is the percentage of the 3m tokens that correspond to the rewardable hours the station provided.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:33]: 3m --> 3M

File: docs/rewards/reward-boosts.mdx [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation) 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").
+
+Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.
+
+Calculation:
+
+The following calculations will take place for each station, the day before the Token Launch:
+
+`RewardableBetaHours[s]`.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:28]: Instead of a dot at the end please use `:`

File: docs/rewards/reward-boosts.mdx [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+This is the total number of hours the station provided valid data during the Beta period.
+A rewardable hour is an hour during which the station sent at least one valid weather data point.
+
+`StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:32]: Add at the end please a `:`

File: docs/rewards/reward-boosts.mdx [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+This is the percentage of the 3m tokens that correspond to the rewardable hours the station provided.
+`DailyBetaRewards[s] = StationBetaRewards[s]/365`
+
+This is the beta rewards allocated daily to this station.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:36]: Please move those in a new line and do the following:

`DailyBetaRewards[s] = StationBetaRewards[s]/365`: This is the beta rewards allocated daily to this station.

File: docs/rewards/reward-boosts.mdx [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+
+:::info
+Stations must have a wallet defined to be part of the Beta Rewards allocation.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:40]: Let's rephrase it to:

> Each user who has a station eligible for beta rewards, must have a wallet defined to be part of the Beta Rewards allocation.

And hyperlink the "wallet" to the "Add/edit wallet address" section in our docs please.

File: docs/rewards/reward-boosts.mdx [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+### Cell Boost
+
+Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell. 
+
+
+#### Examples of reward programs that may be implemented using **Rewards Boost**, in the near future are:
+
+
+- As part of the &&Beta Rewards** program, the first 100 devices that join the network will receive tokens.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:55]: &&Beta Rewards** --> **Beta Rewards**

File: docs/rewards/reward-boosts.mdx [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+:::note
+The Beta Rewards allocation is managed by the [WeatherXM Network](https://weatherxm.network).
+:::
+
+### Cell Boost
+
+Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell. 
+
+
+#### Examples of reward programs that may be implemented using **Rewards Boost**, in the near future are:
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:52]: Let's rephrase it to:

> Examples of reward boosts that may be implemented in the near future are:

And instead of `####` use `###`

File: docs/rewards/reward-boosts.mdx [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+
+- As part of the &&Beta Rewards** program, the first 100 devices that join the network will receive tokens.
+- Backfill of rewards that were not allocated to certain devices due to malfunctions in the Quality of Data (QoD) mechanism](/project/quality-of-data) or [Proof of Location (PoL) mechanism](/project/proof-of-location).
+- Public Goods Funding (PGF) for devices that are deployed and contribute to the network in a humanitarian way, such as providing data in an underserved area or during a specific time. 
+- Devices that contribute to the network during a specific event, such as a natural disaster, will receive a boost in rewards.
+- Demand driven rewards, a third party who needs data from specific cells.
+
+:::tip
+The **Rewards Boost** is a mechanism that can be used to incentivize specific behavior from devices. It is a flexible mechanism that can be used to incentivize any behavior that is beneficial to the network.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:62]: Rewards Boost -> Reward Boosts

File: docs/rewards/reward-boosts.mdx [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+:::
+
+:::info
+All the above are **examples** of reward programs that may be implemented using the Rewards Boost mechanism.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:66]: Let's rephrase it to:

> All the above are **examples** of reward boosts that may be implemented.

File: docs/rewards/reward-boosts.mdx [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+id: reward-boosts
+title: Rewards Boosts
+sidebar_label: Rewards Boosts
+slug: /project/rewards-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+## Overview
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:11]: Let's use `###` please for better structure below.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

**Review by PavlosTze — APPROVED** [review]

File: docs/rewards/reward-boosts.mdx [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+**Reward boosts** are additional rewards given to devices that contribute weather data during daily distribution, added to their **base rewards**. Learn more about the daily rewards at the [Reward Mechanism (RM) ](/reward-mechanism) section.
+
+**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 
+
+So far the following **Reward Boosts** are sheduled to be implemented:
+
+### Beta Rewards
+
+According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation), 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:21]:
```suggestion
According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation), 3M $WXM are reserved to reward stations that participated to the network before the Mainnet Launch Day ("Beta period").
```

File: docs/rewards/reward-boosts.mdx [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+
+Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.
+
+#### Calculation:
+
+The following calculations will take place for each station, the day before the Token Launch:
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:27]:
```suggestion
The following calculations will take place for each station, the day before the Mainnet Launch:
```

File: docs/rewards/reward-boosts.mdx [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+
+`RewardableBetaHours[s]`:
+This is the total number of hours the station provided valid data during the Beta period.
+A rewardable hour is an hour during which the station sent at least one valid weather data point.
+
+`StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`:
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:33]: This syntax is somewhat confusing. I strongly suggest we go with Math formula syntax, as described here:
https://github.blog/2022-05-19-math-support-in-markdown/

The content could be take - and adapted - from here our internal outline wiki.

File: docs/rewards/reward-boosts.mdx [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+Each user who has a station eligible for beta rewards, must have a wallet defined to be part of the Beta Rewards allocation. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
+:::
+
+:::note
+The Beta Rewards allocation is managed by the [WeatherXM Network](https://weatherxm.network).
+:::
+
+### Cell Boost
+
+Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell.
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:51]:
```suggestion
Cell Boost is a mechanism that can be used to incentivize weather station deployments on a specific cells / areas of interest.
```

File: docs/rewards/reward-boosts.mdx [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+### Cell Boost
+
+Cell Boost is a mechanism that can be used to incentivize waether stations to be deployed on a specific cell. 
+
+
+### Examples of reward programs that may be implemented using **Rewards Boost**, in the near future are:
+
+
+- As part of the **Beta Rewards** program, the first 100 devices that join the network will receive tokens.
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:57]: This is just an idea, not to be publicly communicated.
```suggestion
```

File: docs/rewards/reward-boosts.mdx [hunk 24]
```diff
@@ -0,0 +1,69 @@
...
+- Backfill of rewards that were not allocated to certain devices due to malfunctions in the [Quality of Data (QoD) mechanism](/project/quality-of-data) or [Proof of Location (PoL) mechanism](/project/proof-of-location).
+- Public Goods Funding, for devices that are deployed and contribute to the network in a humanitarian way, such as providing data in an underserved area or during a specific time. 
+- Devices that contribute to the network during a specific event, such as a natural disaster, will receive a boost in rewards.
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:60]:
```suggestion
- Devices that contribute to the network during a specific event, such as a natural disaster, may receive a boost in rewards.
```

File: docs/rewards/reward-boosts.mdx [hunk 25]
```diff
@@ -0,0 +1,69 @@
...
+- Demand driven rewards, a third party who needs data from specific cells.
```
Comment (eeVoskos) [inline docs/rewards/reward-boosts.mdx:61]: This is the cell boost.
```suggestion
```

**Review by eeVoskos — CHANGES_REQUESTED** [review]

File: docs/rewards/reward-boosts.mdx [hunk 26]
```diff
@@ -0,0 +1,69 @@
...
+sidebar_label: Reward Boosts
+slug: /project/reward-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+### Overview
+
+**Reward boosts** are additional rewards given to devices that contribute weather data during daily distribution, added to their **base rewards**. Learn more about the daily rewards at the [Reward Mechanism (RM) ](/reward-mechanism) section.
```
[the remark this answers is not in the document]
Comment (ipanagiv) [reply docs/rewards/reward-boosts.mdx:13]: added.

File: docs/rewards/reward-boosts.mdx [hunk 27]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_label: Reward Boosts
+slug: /project/reward-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+### Overview
+
+**Reward boosts** are additional rewards given to stations that contribute weather data during daily distribution,in addition to the [**base rewards**](/reward-mechanism#daily-rewards-calculation-process). Learn more about the daily rewards in the [Reward Mechanism (RM) ](/reward-mechanism) section.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:13]: daily distribution,in addition to the -----> daily distribution, in addition to the

File: docs/rewards/reward-boosts.mdx [hunk 28]
```diff
@@ -0,0 +1,67 @@
...
+
+**Reward Boosts** rewards stations with extra $WXM tokens for meeting specific criteria.
```
Comment (PavlosTze) [inline docs/rewards/reward-boosts.mdx:15]: rewards stations ---> reward stations

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/rewards/reward-boosts.mdx (hunk 28 above)
Comment (PavlosTze) [reply docs/rewards/reward-boosts.mdx:15]: Fix this also @ipanagiv please!

**Review by PavlosTze — APPROVED** [review]

**Review by eeVoskos — APPROVED** [review]
